--- basemap-1.1.0.orig/lib/mpl_toolkits/basemap/__init__.py	2017-05-04 13:53:42.000000000 -0700
+++ basemap-1.1.0/lib/mpl_toolkits/basemap/__init__.py	2018-02-12 10:36:13.000000000 -0800
@@ -1296,7 +1296,7 @@
                 # numpy array (first column is lons, second is lats).
                 polystring = bdatfile.read(bytecount)
                 # binary data is little endian.
-                b = np.array(np.fromstring(polystring,dtype='<f4'),'f8')
+                b = np.array(np.frombuffer(polystring,dtype='<f4'),'f8')
                 b.shape = (npts,2)
                 b2 = b.copy()
                 # merge polygons that cross dateline.
@@ -5174,7 +5174,7 @@
     gzip.open(os.path.join(basemap_datadir,'lsmask_%smin_%s.bin' %\
         (grid,resolution)), 'rb')
     lsmask =\
-    np.reshape(np.fromstring(lsmaskf.read(),dtype=np.uint8),(nlats,nlons))
+    np.reshape(np.frombuffer(lsmaskf.read(),dtype=np.uint8),(nlats,nlons))
     if lakes:
         lsmask =\
         np.where(lsmask==2,np.array(0,dtype=np.uint8),lsmask)
