--- basemap-1.2.1rel.orig/lib/mpl_toolkits/basemap/__init__.py	2019-08-07 22:03:01.000000000 -0300
+++ basemap-1.2.1rel/lib/mpl_toolkits/basemap/__init__.py	2019-08-08 11:15:02.145868394 -0300
@@ -53,15 +53,10 @@
 import _geoslib
 import functools
 
-# basemap data files now installed in lib/matplotlib/toolkits/basemap/data
-# check to see if environment variable BASEMAPDATA set to a directory,
-# and if so look for the data there.
-if 'BASEMAPDATA' in os.environ:
-    basemap_datadir = os.environ['BASEMAPDATA']
-    if not os.path.isdir(basemap_datadir):
-        raise RuntimeError('Path in environment BASEMAPDATA not a directory')
+if os.name == 'nt':
+    basemap_datadir = os.path.join(sys.prefix, 'Library', 'share', 'basemap')
 else:
-    basemap_datadir = os.sep.join([os.path.dirname(__file__), 'data'])
+    basemap_datadir = os.path.join(sys.prefix, 'share', 'basemap')
 
 __version__ = '1.2.1'
 
@@ -154,7 +149,8 @@
              }
 
 # create dictionary that maps epsg codes to Basemap kwargs.
-epsgf = open(os.path.join(basemap_datadir, 'epsg'))
+pyproj_datadir = os.environ['PROJ_LIB']
+epsgf = open(os.path.join(basemap_datadir,'epsg'))
 epsg_dict={}
 for line in epsgf:
     if line.startswith("#"):
@@ -1222,11 +1218,12 @@
         read boundary data, clip to map projection region.
         """
         msg = dedent("""
-        Unable to open boundary dataset file. Only the 'crude', 'low',
-        'intermediate' and 'high' resolution datasets are installed by default.
-        If you are requesting a 'full' resolution dataset, you may need to
-        download and install those files separately
-        (see the basemap README for details).""")
+        Unable to open boundary dataset file. Only the 'crude' and  'low',
+        resolution datasets are installed by default.
+        If you are requesting an, 'intermediate', 'high' or 'full'
+        resolution dataset, you may need to download and install those
+        files separately with
+        `conda install basemap-data-hires`.""")
         # only gshhs coastlines can be polygons.
         if name != 'gshhs': as_polygons=False
         try:
@@ -1981,6 +1978,11 @@
         """
         ax = ax or self._check_ax()
         gis_file = os.path.join(basemap_datadir,'UScounties')
+        if not os.path.isfile(gis_file + '.shp'):
+            msg = ('Cannot find {}.shp\nYou can install it with'
+                   '`conda install basemap-data-hires`').format
+            raise IOError(msg(gis_file))
+        ax = ax or self._check_ax()
         county_info = self.readshapefile(gis_file,'counties',\
                       default_encoding='latin-1',drawbounds=drawbounds)
         counties = [coords for coords in self.counties]
