diff -ru basemap-1.3.6.orig/packages/basemap/src/mpl_toolkits/basemap/__init__.py basemap-1.3.6/packages/basemap/src/mpl_toolkits/basemap/__init__.py
--- basemap-1.3.6.orig/packages/basemap/src/mpl_toolkits/basemap/__init__.py	2023-02-25 11:30:44.690631200 +0100
+++ basemap-1.3.6/packages/basemap/src/mpl_toolkits/basemap/__init__.py	2023-02-25 11:31:22.559285100 +0100
@@ -1226,7 +1226,8 @@
         Unable to open boundary dataset file. Only the 'crude', 'low' and
         'intermediate' resolution datasets are installed by default. If you
         are requesting a 'high' or 'full' resolution dataset, you need to
-        install the `basemap-data-hires` package.""")
+        install the `basemap-data-hires` package with
+        `conda install -c conda-forge basemap-data-hires`.""")
         # only gshhs coastlines can be polygons.
         if name != 'gshhs': as_polygons=False
         try:
