--- basemap-1.1.0.orig/lib/mpl_toolkits/basemap/__init__.py	2017-05-04 17:53:42.000000000 -0300
+++ basemap-1.1.0/lib/mpl_toolkits/basemap/__init__.py	2018-08-31 13:09:10.768728089 -0300
@@ -140,7 +140,8 @@
              }
 
 # create dictionary that maps epsg codes to Basemap kwargs.
-epsgf = open(os.path.join(pyproj.pyproj_datadir,'epsg'))
+pyproj_datadir = os.environ['PROJ_LIB']
+epsgf = open(os.path.join(pyproj_datadir,'epsg'))
 epsg_dict={}
 for line in epsgf:
     if line.startswith("#"):
