diff -ru basemap-1.3.9.orig/packages/basemap/src/mpl_toolkits/basemap/__init__.py basemap-1.3.9/packages/basemap/src/mpl_toolkits/basemap/__init__.py
--- basemap-1.3.9.orig/packages/basemap/src/mpl_toolkits/basemap/__init__.py	2023-12-26 11:45:00
+++ basemap-1.3.9/packages/basemap/src/mpl_toolkits/basemap/__init__.py	2023-12-26 11:45:00
@@ -1228,7 +1228,8 @@
         Unable to open boundary dataset file. Only the 'crude', 'low' and
         'intermediate' resolution datasets are installed by default. If you
         are requesting a 'high' or 'full' resolution dataset, you need to
-        install the `basemap-data-hires` package.""")
+        install the `basemap-data-hires` package with
+        `conda install -c conda-forge basemap-data-hires`.""")
         # only gshhs coastlines can be polygons.
         if name != 'gshhs': as_polygons=False
         try:
